### PR TITLE
Update quality_control.yml

### PR DESCRIPTION
**Pull request by WillBanks1**
Fixed Requirements.txt bug

### CODE REVIEW

File: .github/workflows/quality_control.yml [hunk 1]
```diff
@@ -22,7 +22,10 @@ jobs:
           python-version: 3.11
       - name: "Install all libraries"
         run: |
-          pip3 install -r requirements.txt
+          pip3 install pytest pylint
+          for FILE in $(find . -name 'requirements.txt'); do
+            pip3 install -r $FILE
+          done
```
Comment (joe1606) [inline .github/workflows/quality_control.yml:28]: Would pylint and pytest would both installed from requirements.txt anyway? not sure if you need to run `pip3 install pytest pylint`

File: .github/workflows/quality_control.yml [hunk 2]
```diff
@@ -22,7 +22,10 @@ jobs:
...
           echo "library installation complete"
       - name: "Run Pytest"
         run: |
```
Comment (joe1606) [inline .github/workflows/quality_control.yml:31]: Just wondering, will pylint run on all python files, including test files?

**Review by joe1606 — COMMENTED** [review]
Looks good overall, just have a couple of questions with the functionality of the script, but as far as I know it'll work

File: .github/workflows/quality_control.yml (hunk 1 above)
Comment (WillBanks1) [reply .github/workflows/quality_control.yml:28]: I was thinking that way we don't need to add pylint and pytest to our requirements files - as the project doesn't actually need them to run. Plus we don't have to repeat pylint and pytest in every requirements file.

**Review by joe1606 — APPROVED** [review]
Will approve first, as we want to test functionality and suggested changes are more QOL improvements

File: .github/workflows/quality_control.yml (hunk 2 above)
Comment (WillBanks1) [reply .github/workflows/quality_control.yml:31]: It will run on all .py files - potential scope later to get it to ignore any files starting with 'test_'

**Pull request merged**